src/algo.py

```diff
@@ -0,0 +1,34 @@
+# triggered by algo-is_prime snippet text
+def is_prime(n: int) -> bool:
+    """
+    Return True if n is prime, False otherwise.
+    """
+    if n < 2:
+        return False
+    elif n < 4:
+        return True
+    elif n % 2 == 0 or n % 3 == 0:
+        return False
+    else:
+        for i in range(5, int(n**0.5)+1, 6):
+            if n % i == 0 or n % (i+2) == 0:
+                return False
+        return True
+
+# triggered by algo-fast_expo snippet text
```

Comment:
Line 18, the name is algo-fast_power

Reply:
You are right, but my concerns were
- Adding `CI` `CD` pipeline for our repo
- Adding src and test directory
- whether you are ok with `pytest` or not

That's the reason i made a pull request so that , i can notify these changes

Reply:
What do u mean ci cd? Auto publish to marketplace? I afraid that will bump the version too much. Like I added 5-10 snippets usually I give myself 1 week think where else to improve 😅
Src and Test sound good. But it need to be done each time a snippet is added thought else too much work to do. For algo snippet sure, pytest will be reliable

Reply:
ok `CI` `CD` are continues integration and continues deployment 

- with `CI` we can ensure if contributor pull request passes our `style guide` existing test case then only we merge it into our repo
- with `CD` as you said automatically deploy our extension

Reply:
yaa fill snippet involves `logic` its best to have both `src` and `test` for it

Reply:
Probably not CI or CD for now 😅 will be hard to keep track without double checking. Like `Line 18` the name is `algo-fast_power` after CI then change again will caused lot of commits history

Reply:
Ok

Reply:
> Ok

Tell me when u finish done commit then i'll merge it